### ISSUE TITLE
Duplicate Instance while Running

### ISSUE THREAD

**Issue by mrezauli** [leokhoa/laragon#167]
I've found a issue that Laragon create a instance when i open it although it has a running instance on my system tray[simply two instance of laragon run simultaneously].
But when I stop/exit from one instance it stops both instance!
**If Laragon prevents running two instance simultaneously it will be helpfuil.**
My environment:
OS: Windows 10 Pro x64
Laragon: v4.0.15

**Comment by svanlaere** [comment]
Yes this really should be fixed since its annoying, allowing a single instance is indeed the solution.
Hope this gets fixed soon.

**Comment by tdchien** [comment]
+1 Agree. Hope will update for this soon
